tests/database_test.cpp

```diff
@@ -43,10 +43,12 @@ class DatabaseTest : public ::testing::Test {
 };
 
 TEST_F(DatabaseTest, Instantiate) {}
+
 TEST_F(DatabaseTest, InstantiateWithConfig) {
   db_.reset(nullptr);
   LineairDB::Config conf;
   conf.checkpoint_period = 1;
+  ASSERT_EQ("./lineairdb_logs", conf.lineairdb_logs_dir);
```

Comment:
It would be unnecessary to test the default configuration with a hard-coded string.